plotter.py

```diff
@@ -0,0 +1,29 @@
+"""
+Plotter functions.
+"""
+
+import pandas as pd
+import plotly
+import plotly.express as px
+
+def line(raw_data, tittle='2D-line',
+         x_axial='X', y_axial='Y', instruments='instruments'):
```

Comment:
Does `intruments` really matter?
Nothing will change as long as we don't let it be something conflicting with `x_axial` or `y_axial`.

Reply:
I was thinking that, user may want to plot more than one line in this picture (ex. drawing pnl for USD and AUD in the same picture). Thus, we need ```instruments``` to  plot these lines.
Hmmm, but without input instrument, the code can do this as well.

Reply:
Whatever its value is, your code always treat it as instruments. So there is no need to set its value at all.